src/components/document-id/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@bcrs-shared-components/document-id",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "axios": "^0.27.2",
+    "@bcrs-shared-components/interfaces": "^1.1.23",
```

Comment:
The new component doesn't depend on interfaces, but it does depend on enums. Small update, please.

![image](https://github.com/user-attachments/assets/6e35230c-c677-4f9c-a3a1-7e800e871b8b)

Reply:
Gracias!